### ISSUE TITLE
Views/template.#.#.#.ejs overwritten when extension is updated

### ISSUE THREAD

**Issue by GrumpyLabRat** [evilz/vscode-reveal#848]
Unless there is a functionality support reason, the template.ejs file in view should persist between versions. It introduces unnecessary debugging when users have built in customizations (i.e. edit the logo placement, or have added additional features to the template.).

**Comment by evilz** [comment]
??

**Comment by GrumpyLabRat** [comment]
In the file path .vscode/extensions/evilz.vscode-reveal-4.1.x/views/ there is a file called template.4.1.3.ejs in version 4.1.1 of the extension.  This template contains the information things like where the logo keyword should be placed, etc.  I editted it because I don't like the logo to show up where it does by default.  I also added the ability to add a second logo (for collaborations, etc.). When the extension version 4.1.2 was pushed last night, this file was overwritten.  I had it backed up, so I was able to correct the issue, but my point (and request) is that this file should not be touched or another location should be provided to allow for customizations as was done for the custom theme capability.
